src/libs/connect-lib.js

```diff
@@ -167,5 +159,89 @@ export async function testConnectors(cluster, service, connectors) {
       console.log(`Testing connector: ${connector.name}`);
       return testConnector(workerIp, connector);
     })
+  ).then((res) => {
+    return res;
+  });
+}
+
+export async function findTaskIp(cluster) {
+  const client = new ECSClient();
+  const { taskArns } = await client.send(
+    new ListTasksCommand({
+      cluster: cluster,
+      desiredStatus: "RUNNING",
+    })
   );
+  if (taskArns.length === 0) {
+    throw `No task found for cluster ${cluster}`;
+  }
+  const tasks = (
+    await client.send(
+      new DescribeTasksCommand({
+        cluster: cluster,
+        tasks: [taskArns[0]],
```

Comment:
could the `taskArns` array ever have more than one item?

Reply:
How about changing this:
```
if (taskArns.length === 0) {
    throw `No task found for cluster ${cluster}`;
  }
```
to
```
if (taskArns.length !== 1) {
    throw `taskArns for cluster ${cluster} is not equal to 1`;
  }
```
or that may be too clunky.  I think we should throw an error if its not 1 item.  So we ensure we're appropriately dealing with multiple items when/if that happens.